### DISCUSSION TITLE
Way to unbind a keymap in neo-tree window?

### DISCUSSION THREAD

**Discussion by xcyu-stripe** [nvim-neo-tree/neo-tree.nvim#1242] [Q&A] [answered]
Is there a way to unbind "/" so that it functions normally and just search for for whats typed in like in a normal buffer?
I tried the following but it didn't change anything:
```lua
    opts = {
      filesystem = {
        window = {
          mappings = {
            ["/"] = nil,
          },
        },
      },
    },
```
Reason I'm asking is that I'd like to use regular `/` to move around in the neo-tree buffer.

**Comment by bew** [comment, marked as answer]
Yes
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/230ff118613fa07138ba579b89d13ec2201530b9/doc/neo-tree.txt#L461-L477

**Comment by wookayin** [comment]
`["key"] = false` also works as well as `["key"] = "noop"`.

```
          mappings = {
            ["/"] = false,
          },
```
